.bldr.toml

```diff
@@ -162,6 +162,10 @@ plan_path = "cppunit"
 plan_path = "cpputest"
 [cuda]
 plan_path = "cuda"
+[cuda9]
```

Comment:
Since your plans required `cuda/plan.sh`, you will need to add these lines to each section:

```
paths = [
  "cuda/*"
]
```